posix/signal.c

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+
+#include <openenclave/corelibc/errno.h>
+#include <openenclave/corelibc/signal.h>
+#include <openenclave/corelibc/stdlib.h>
+#include <openenclave/corelibc/unistd.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/posix/lock.h>
+#include <openenclave/internal/posix/raise.h>
+#include <openenclave/internal/print.h>
+#include <openenclave/internal/thread.h>
+#include <openenclave/internal/trace.h>
+#include "posix_t.h"
+
+// Poll uses much of the infrastructure from epoll.
+
+static void _handle_ignore(int signum);
+
+static struct oe_sigaction _actions[__OE_NSIG] = {{{0}}};
+
+static oe_sighandler_t _default_actions[__OE_NSIG] = {
+    _handle_ignore, _handle_ignore, _handle_ignore, _handle_ignore,
```

Comment:
>_handle_ignore [](start = 4, length = 14)

It seems really bad that the default action for for all signals is to ignore ... shouldn't this always rethrow as the default (i.e. we should define `_handle_default(int signum)`

Reply:
Removed singal feature from this PR.